Tests/LibWeb/Text/input/navigation/history-replace-push-state-race.html

```diff
@@ -0,0 +1,12 @@
+<script src="../include.js"></script>
+<script>
+    asyncTest(done => {
```

Comment:
i am not very sure it has to be an `asyncTest`, can look into that

Reply:
this test has to be async, if it is just a test() it seems to not produce anything 
```
--- ../../Tests/LibWeb/Text/expected/./navigation/history-replace-push-state-race.txt
+++ ../../Tests/LibWeb/Text/expected/./navigation/history-replace-push-state-race.txt
@@ -1 +0,0 @@
-test done!
```

if history replace calls (one or both) are removed a sync test works,
i dont know enough about how History replaceState() behaves to know why yet